javascript/extractor/src/com/semmle/js/extractor/ASTExtractor.java

```diff
@@ -1806,6 +1806,9 @@ public Label visit(ImportSpecifier nd, Context c) {
       Label lbl = super.visit(nd, c);
       visit(nd.getImported(), lbl, 0, IdContext.LABEL);
       visit(nd.getLocal(), lbl, 1, c.idcontext);
```

Comment:
To be consistent with how we handle `import type {...}` declarations I believe the `c.idcontext` above should be changed to,
```java
nd.hasTypeKeyword() ? IdContext.TYPE_ONLY_IMPORT : c.idcontext
````
This rarely matters since TypeScript complains about attempts to reference the imported name as a variable, but we shouldn't handle the two cases differently.